projects/plugins/jetpack/modules/likes/queuehandler.js

```diff
@@ -283,13 +283,14 @@ function jetpackLoadLikeWidgetIframe( wrapperID ) {
 	if ( placeholder.hasClass( 'post-likes-widget-placeholder' ) ) {
 		var postLikesFrame = document.createElement( 'iframe' );
 
-		postLikesFrame[ 'class' ] = 'post-likes-widget jetpack-likes-widget';
+		postLikesFrame.classList.add( 'post-likes-widget', 'jetpack-likes-widget' );
 		postLikesFrame.name = $wrapper.data( 'name' );
 		postLikesFrame.src = $wrapper.data( 'src' );
 		postLikesFrame.height = '18px';
 		postLikesFrame.width = '200px';
```

Comment:
What about setting initially these
```suggestion
		postLikesFrame.height = '55px';
		postLikesFrame.width = '100%';
```
and removing the else part?

Reply:
Thank you for your suggestion. I've updated the code. 👍